src/project_variables.rs

```diff
@@ -92,41 +95,38 @@ const RESERVED_NAMES: [&str; 7] = [
 ];
 
 pub fn fill_project_variables<F>(
-    mut template_object: Object,
-    template_config: &Config,
+    template_object: &mut Object,
```

Comment:
reference to the previous comment.

Here it's done exactly this way, mutable input, no return `Object` anylonger.

Reply:
but here, the API does *not* need the ownership